cloud_images/centos7/dcos_vol_setup.sh

```diff
@@ -35,23 +35,48 @@ function main {
     exit 1
   fi
 
+  echo "Waiting for $device to come online"
+  until test -b "$device"; do sleep 1; done
```

Comment:
I had that in at first but dropped it. It was just adding noise to the log and the journal is timestamped anyways so can tell how long it took by the delta between this and the next echo.